tools/start_server.py

```diff
@@ -273,15 +273,32 @@ def handle_start_video_stream(self, _method, _params):
     if 'url' not in _params:
       return self.gen_error_msg('Missing mandatory param \'url\'')
 
-    server_endpoint = 'http://' + Flags.FILE_SERVER_HOST + ':' + str(Flags.VIDEO_SERVER_PORT)
-    ffmpeg_process = ffmpeg.input(_params['url']).output(server_endpoint, vcodec='vp8', format='webm', listen=1, multiple_requests=1).run_async(pipe_stderr=True)
-    o = pexpect.fdpexpect.fdspawn(ffmpeg_process.stderr.fileno(), logfile=sys.stdout.buffer)
-    index = o.expect(["Input", pexpect.EOF, pexpect.TIMEOUT])
-
-    if index != 0:
-      return self.gen_error_msg('Streaming data not available from SDL')
-
-    return { 'success': True, 'params': { 'endpoint': server_endpoint } }
+    if 'config' not in _params:
+      return self.gen_error_msg('Missing mandatory param \'config\'')
+
+    if _params['config']['protocol'] == 'RTP':
+      print('\033[33mFFMpeg does not support RTP video\033[0m')
```

Comment:
Say: SDL does not support in browser RTP video

FFMPEG says it supports RTP, we just cant get it to work :(

Reply:
right, its RTP over TCP it doesn't support. comment updated in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/14fa5705a549a868d956d456fe3c68d51f1ed14f